test/integration/good/code-gen/cl.expected

```diff
@@ -427,7 +423,7 @@ class optimize_glm_model final : public model_base_crtp<optimize_glm_model> {
       y_vi_d = std::vector<int>(n, std::numeric_limits<int>::min());
       
       current_statement__ = 189;
-      assign(y_vi_d, context__.vals_i("y_vi_d"), "assigning variable y_vi_d");
+      y_vi_d = context__.vals_i("y_vi_d");
```

Comment:
These go through assign on master but I am guessing they dont have to as well?

Reply:
Essentially I think anywhere we were doing an `assign()` without any form of indexing we can just go right through to an `=`

Reply:
Not really, only when we we know size of lhs is equal to rhs (see https://github.com/stan-dev/stanc3/issues/523). With context__ and in__ I think we can avoid it right?